src/silo_api/database_directory_watcher.cpp

```diff
@@ -150,7 +155,7 @@ void silo_api::DatabaseDirectoryWatcher::checkDirectoryForData(Poco::Timer& /*ti
       }
    }
    SPDLOG_INFO(
-      "New database with version {} successfully loaded.",
+      "Did not load new database with version {} successfully.",
```

Comment:
We have log statements in every `catch`. Should we maybe simply add a clarifying "failed to load new database with version xyz" to the `SPDLOG_ERROR(ex.what());`?